src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -450,15 +450,17 @@ class SceneryWindow final : public Window
                 }
                 else if (tabSelectedScenery.SceneryType == SCENERY_TYPE_WALL)
                 {
-                    gCurrentToolId = static_cast<Tool>(GetWallEntry(tabSelectedScenery.EntryIndex)->tool_id);
+                    gCurrentToolId = static_cast<Tool>(
+                        OpenRCT2::ObjectManager::GetObjectEntry<WallSceneryEntry>(tabSelectedScenery.EntryIndex)->tool_id);
```

Comment:
Isn't the use of the ```OpenRCT2::``` namespace specifier before all ```ObjectManager``` calls redudent in this file because of line 38?

Reply:
Yeah i need to go through and remove the namespace from the files that are already in the namespace.